model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/SingleEntityCredentialManagerCacheAdapter.java

```diff
@@ -0,0 +1,119 @@
+package org.keycloak.models.cache.infinispan;
```

Comment:
Missing header - same is true for other files being added in this PR

Reply:
Addressed in upcoming commit https://github.com/ahus1/keycloak/commit/1cf8f81cd4b7f813abee3837c3d9a83c3f0c0f78

Reply:
Fixed in this PR commit https://github.com/keycloak/keycloak/commit/7e21088b3875f92493fc215a623b6c417ba82379